lib/nerve/reporter/base.rb

```diff
@@ -25,6 +25,17 @@ def update_data(new_data='')
     def ping?
     end
 
+    def get_service_data(service)
+      %w{instance_id host port}.each do |required|
+        raise ArgumentError, "missing required argument #{required} for new service watcher" unless service[required]
+      end
+      d = {'host' => service['host'], 'port' => service['port'], 'name' => service['instance_id']}
+      if service['weight'].to_i > 0
```

Comment:
Is setting weight to 0 something that we want to support? It seems like it would be nice?